gcc/rust/hir/rust-ast-lower-implitem.h

```diff
@@ -54,15 +54,19 @@ class ASTLowerImplItem : public ASTLoweringBase
 
   void visit (AST::MacroInvocation &invoc) override
   {
-    if (!invoc.has_semicolon ())
-      return;
-
-    AST::ASTFragment &fragment = invoc.get_fragment ();
-
-    // FIXME
-    // this assertion might go away, maybe on failure's to expand a macro?
-    rust_assert (!fragment.get_nodes ().empty ());
-    fragment.get_nodes ().at (0).accept_vis (*this);
+    // if (!invoc.has_semicolon ())
+    //   return;
+
+    // AST::ASTFragment &fragment = invoc.get_fragment ();
+
+    // // FIXME
+    // // this assertion might go away, maybe on failure's to expand a macro?
+    // rust_assert (!fragment.get_nodes ().empty ());
+    // fragment.get_nodes ().at (0).accept_vis (*this);
+    rust_fatal_error (
```

Comment:
let's keep these as fatal_errors or assertions until the end of the macro expansion milestone since there should be errors already emitted prior to this that allow us to ignore visiting these in the first place.